backend/src/routes/recipes.ts

```diff
@@ -0,0 +1,42 @@
+import express, { Request, Response, Router } from 'express';
+import { readDatabase, writeDatabase, Recipe } from '../database/dbService';
+
+const router: Router = express.Router();
+router.get('/', async (req: Request, res: Response) => {
+    const db = await readDatabase();
+    res.json(db.recipes);
+});
+
+router.post('/', async (req: Request, res: Response) => {
+    const { title, ingredients, steps } = req.body;
+    const newRecipe: Recipe = { id: Date.now().toString(), title, ingredients, steps };
+
+    const db = await readDatabase();
+    db.recipes.push(newRecipe);
+    await writeDatabase(db);
+    res.status(201).json(newRecipe);
```

Comment:
I recommend using https://www.npmjs.com/package/http-status-codes for all the status codes in this project